### PR TITLE
Update version to 2024.07.08-3

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

File: changelog-2024.07.08-3.patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+@@ -111,6 +111,25 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2024.07.08-3" date="2024-07-08">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>Better address search in the USA based on US Census TIGER data</li>
++           <li>Fixed rendering of texts in Indic, Arabic, Thai, Lao, Khmer, Myanmar, Tibetan, Hangul, Hebrew scripts</li>
```
Comment (biodranik) [inline changelog-2024.07.08-3.patch:14]: - Better address search in the USA based on Topologically Integrated Geographic Encoding and Referencing system (TIGER) data. Map size for the USA has increased from 10Gb to 13Gb (including Wikipedia and elevation data)
- Fixed rendering of texts in Indic (covering Devanagari, Bengali, Gujarati, Gurmukhi, Kannada, Malayalam, Oriya, Tamil, Telugu), Arabic (N'Ko, Syriac, Mongolian), Thai and Lao, Khmer, Myanmar, Tibetan, Hangul, Hebrew scripts
- New Malayalam and Bengali fonts

File: changelog-2024.07.08-3.patch [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
++           <li>OSM data as of July 2, 2024</li>
++           <li>Display major roads earlier on the world overview map</li>
```
Comment (biodranik) [inline changelog-2024.07.08-3.patch:16]: - A selected place now shows more types/categories/amenities

Comment (biodranik) [reply changelog-2024.07.08-3.patch:16]: - Display good quality fine gravel pedestrian paths as white dashed
- Support more road/path surface types (bricks, stone, grass paver, soil)
- Mountain search also shows volcanoes
- Better search results for faraway places
- Search now finds places with a number in their names
- Find parcel lockers using their ref numbers

File: changelog-2024.07.08-3.patch (hunk 1 above)
Comment (Ferenc-) [reply changelog-2024.07.08-3.patch:14]: Done

File: changelog-2024.07.08-3.patch (hunk 2 above)
Comment (Ferenc-) [reply changelog-2024.07.08-3.patch:16]: Done

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**